test/wallet/load_non-complaint_ERC20_test.go

```diff
@@ -167,48 +169,58 @@ var _ = Describe("wallet load non-compliant ERC20", func() {
 
 			}) //equal to approval
 
-			When("a bigger amount than the one owned is tried to be transfered ", func() {
+			When("a bigger amount than the one owned is loaded", func() {
 
 				It("Should revert", func() {
 					tx, err := Wallet.LoadTokenCard(Owner.TransactOpts(ethertest.WithGasLimit(300000)), NonCompliantERC20Address, big.NewInt(1001))
 					Expect(err).ToNot(HaveOccurred())
 					Backend.Commit()
-					Expect(isGasExhausted(tx, 300000)).To(BeFalse())
 					Expect(isSuccessful(tx)).To(BeFalse())
+					returnData, _ := ethCall(tx)
+					Expect(string(returnData[len(returnData)-64:])).To(ContainSubstring("SafeERC20: low-level call failed"))
 				})
 
 			}) //more than owned (and hence can be approved)
 
-			When("a bigger amount than daily Load limit is tried to be loaded", func() {
+			When("a bigger amount than MAX daily Load limit is loaded", func() {
 
-				//change the exchange rate to be equal to daily Load limit + 1 wei
 				BeforeEach(func() {
-
-					limPlusOneWei, err := Wallet.LoadLimitValue(nil)
-					Expect(err).ToNot(HaveOccurred())
-
-					limPlusOneWei.Add(limPlusOneWei, big.NewInt(1))
-
+					//rate is 1 token  => 1 ETH + 1 wei
 					tx, err := TokenWhitelist.UpdateTokenRate(
 						ControllerAdmin.TransactOpts(),
 						NonCompliantERC20Address,
-						limPlusOneWei,
+						big.NewInt(int64(math.Pow10(18))),
+						big.NewInt(20180913153211),
+					)
+					Expect(err).ToNot(HaveOccurred())
+					Backend.Commit()
+					Expect(isSuccessful(tx)).To(BeTrue())
+
+					// stablecoin rate is 10-6
+					tx, err = TokenWhitelist.UpdateTokenRate(
+						ControllerAdmin.TransactOpts(),
+						StablecoinAddress,
+						big.NewInt(int64(0.007165*math.Pow10(18))),
 						big.NewInt(20180913153211),
 					)
 					Expect(err).ToNot(HaveOccurred())
 					Backend.Commit()
 					Expect(isSuccessful(tx)).To(BeTrue())
 				})
 
-				It("Should revert", func() {
-					//1 token  = 101 eth + 1 wei => it should exceed the daily limit, 1 token =  1*magnitude (10^18)
-					tx, err := Wallet.LoadTokenCard(Owner.TransactOpts(ethertest.WithGasLimit(100000)), NonCompliantERC20Address, EthToWei(1))
+				FIt("Should revert", func() {
```

Comment:
we should remove the `F` here

Reply:
it is WIP so you can see where the problems are